src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,231 @@
+    private String[] users = {"user1|password1", "user2|password2", "user3|password3"};
+
     public static void main(String[] args) {
+        try {
+            new Application().simpleSQL();
+        } catch (ClassNotFoundException | SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void simpleSQL() throws ClassNotFoundException, SQLException {
+        connection = getConnection();
+        deleteTable("user");
+        System.out.println(getTablesName());
+        createTable("user");
+        System.out.println(getTablesName());
+        addUsers(users);
+        showTableData("user");
+        changeUserName("user2", "userсhange1");
+        showTableData("user");
+        deleteUser("user3");
+        showTableData("user");
+        connection.close();
+    }
+
+    /**
+     * Method deletes the user with name "userName" from the table "user".
+     *
+     * @param userName the username you want to delete.
+     * @throws SQLException if a database access error occurs
+     */
+    private void deleteUser(String userName) throws SQLException {
+        if (userName != null) {
+            try (Statement statement = connection.createStatement()) {
+                String query = String.format("DELETE FROM \"user\" WHERE name = '%s'", userName);
+                statement.executeUpdate(query);
+            }
+        }
+    }
+
+    /**
+     * Method changes the username in the "name" field in the "user" table.
+     *
+     * @param oldName old user name. It will be overwritten with the new value.
+     * @param newName new user name value.
+     * @throws SQLException if a database access error occurs
+     */
+    private void changeUserName(String oldName, String newName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("UPDATE \"user\" SET name = '%s' WHERE name = '%s'", newName, oldName);
+            statement.executeUpdate(query);
+        }
+    }
+
+    /**
+     * The method prints the table data to the console in the format Field1 | Field2 | ... | FieldN
+     *
+     * @param tableName the name of the table whose data should be output to the console.
+     * @throws SQLException if a database access error occurs
+     */
+    private void showTableData(String tableName) throws SQLException {
+        if (tableName != null) {
+            String tableData = getTableData(tableName);
+            System.out.println(tableData);
+        } else {
+            System.err.println("Can't display data. Table Name is Null");
+        }
+    }
+
+    /**
+     * The method returns all data from the table.
+     *
+     * @param tableName the name of the table whose data should be returned
+     * @return all data from the table in the format Field1 | Field2 | ... | FieldN
+     * @throws SQLException if a database access error occurs
+     */
+    private String getTableData(String tableName) throws SQLException {
```

Comment:
Just FYI - http://bit.ly/2hWi2Tn... I'm not yet ready for almost all stuffs from Bugaenko but with naming of methods I totally agree with him. Finally it's up to you to agree with that philosophy or not )

Reply:
I'll think about it, but now I'd like to leave it as it is.
Thanks for the link.